app/js/map/loadGMapsApi.provider.js

```diff
@@ -0,0 +1,65 @@
+(function() {
+  'use strict';
+  angular.module('cityInfo.providers').provider('LoadGoogleMapsApi', LoadGoogleMapsApiProvider);
+
+  function LoadGoogleMapsApiProvider() {
+      var myConfig = {
+        baseUrl: 'https://maps.googleapis.com/maps/api/js',
+        apiKey: null,
+        libraries: null,
+        language: 'en'
+      };
+
+      this.setConfig = function(config) {
```

Comment:
Is there some reason you are not binding `this` to a variable here (I see you do it on the other modules)?